github-auth-hook.js

```diff
@@ -34,15 +34,19 @@ passport.use(
                 function(error, response, body) {
                     if (error) {
                         console.error('access to GH org failed:', error);
-                        return done(error);
+                        return done(null, false, { message: error });
```

Comment:
why did you need to change the args in this call to `done` ?

Reply:
bc I'd like to have a consistent way of displaying error in the field message sot that the UI can display them all in the same manner.